src/registries/menus/items/sheet_menu_itemps.ts

```diff
@@ -0,0 +1,84 @@
+import { buildSheetLink, markdownLink } from "../../../helpers";
+import { _lt } from "../../../translation";
+import { MenuItemSpec } from "../../menu_items_registry";
+
+export const linkSheetMenuItem: MenuItemSpec = {
```

Comment:
Not sure it's worth to separate into 2 files `MenuItems` and `MenuRegistry` for the menu items that are only used once. It's a bit more annoying to use.

Reply:
Will people make the effort to go through all the registries ( in addition to all the menu items) to find if what he is developing does not already exist ? :p